common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -184,6 +194,68 @@ def get_cohort_student_input_field_value(self):
             css=self._bounded_selector("#cohort-management-group-add-students")
         ).results[0].get_attribute("value")
 
+    def get_all_content_groups(self):
+        """
+        Returns all the content groups available for associating with the cohort currently being edited.
+        """
+        select = self.q(css=self._bounded_selector(self.content_group_selector))
+        groups = []
+        for option in select:
+            if option.text != "":
+                groups.append(option.text)
+
+        return groups
+
+    def get_cohort_associated_content_group(self):
+        """
+        Returns the content group associated with the cohort currently being edited.
+        If no content group is associated, returns None.
+        """
+        self.select_cohort_settings()
+        radio_button = self.q(css=self._bounded_selector(self.no_content_group_button)).results[0]
+        if radio_button.is_selected():
+            return None
+        option_selector = self.q(css=self._bounded_selector(self.content_group_selector))
+        return option_selector.filter(lambda el: el.is_selected())[0].text
+
+    def set_cohort_associated_content_group(self, content_group=None, select_settings=True):
+        """
+        Sets the content group associated with the cohort currently being edited.
+        If content_group is None, un-links the cohort group from any content group.
+        Presses Save to update the cohort's settings.
+        """
+        if select_settings:
+            self.select_cohort_settings()
+        if content_group is None:
+            self.q(css=self._bounded_selector(self.no_content_group_button)).first.click()
+        else:
+            self._select_associated_content_group(content_group)
+        self.q(css=self._bounded_selector("div.form-actions .action-save")).first.click()
+
+    def _select_associated_content_group(self, content_group):
+        """
+        Selects the specified content group from the selector. Assumes that content_group is not None.
+        """
+        option_selector = self.q(css=self._bounded_selector(self.content_group_selector))
+        option_selector.filter(lambda el: el.text == content_group).first.click()
```

Comment:
One of the Solutions PRs I reviewed had a helper method for dealing with selects. I should see about bringing that into this PR.